jbpm-test-coverage/src/test/java/org/jbpm/test/functional/task/ProcessHumanTaskTest.java

```diff
@@ -83,6 +91,39 @@ public void testProcess() {
 		assertProcessInstanceNotActive(processInstance.getId(), ksession);
 	}
 	
+    @Test
+    public void testProcessWithHumanTaskAndCustomStrategy() {
```

Comment:
I would add `assignment` so it's a little clearer but up to you
```suggestion
    public void testProcessWithHumanTaskAndCustomAssigmentStrategy() {
```

Reply:
done